client/src/components/App/Content/InvestigationForm/useInvestigationForm.ts

```diff
@@ -17,25 +17,39 @@ import {landingPageRoute} from 'Utils/Routes/Routes';
 import {setCities} from 'redux/City/cityActionCreators';
 import { setCountries } from 'redux/Country/countryActionCreators';
 import InvestigationStatus from 'models/enums/InvestigationStatus';
-import useExposuresSaving from "Utils/ControllerHooks/useExposuresSaving";
+import useExposuresSaving from 'Utils/ControllerHooks/useExposuresSaving';
 import { setContactType } from 'redux/ContactType/contactTypeActionCreators';
 
 import useStyles from './InvestigationFormStyles';
 import { defaultTab, tabs } from './TabManagement/TabManagement';
+import { LandingPageTimer } from './InvestigationInfo/InvestigationInfoBar';
+import useContactQuestioning from './TabManagement/ContactQuestioning/useContactQuestioning';
 import { useInvestigationFormOutcome, useInvestigationFormParameters  } from './InvestigationFormInterfaces';
 import { otherSymptomFieldName, otherBackgroundDiseaseFieldName } from './TabManagement/ClinicalDetails/ClinicalDetails';
 
 const useInvestigationForm = (parameters: useInvestigationFormParameters): useInvestigationFormOutcome => {
-    const {clinicalDetailsVariables, personalInfoData, exposuresAndFlightsVariables} = parameters;
+    const { clinicalDetailsVariables, personalInfoData, exposuresAndFlightsVariables, interactedContactsState } = parameters;
 
     const {saveExposureAndFlightData} = useExposuresSaving(exposuresAndFlightsVariables);
+    const { saveContactQuestioning } = useContactQuestioning({ interactedContactsState, setCurrentInteractedContact: () => {} });
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
     const investigatedPatientId = useSelector<StoreStateType, number>(state => state.investigation.investigatedPatientId);
-   
+
+    const classes = useStyles({});
     let history = useHistory();
+
     const [currentTab, setCurrentTab] = useState<Tab>(defaultTab);
+    const [areThereContacts, setAreThereContacts] = useState<boolean>(false);
 
-    const classes = useStyles({});
+    useEffect(() => {
+        initializeTabShow();
+    }, []);
+
+    const initializeTabShow = () => {
+        axios.get('/contactedPeople/' + epidemiologyNumber).then((result: any) => {
+            setAreThereContacts(result?.data?.data?.allContactedPeople?.nodes?.length > 0);
```

Comment:
display swal on fail? (more of a ux question tbh 🤔 )

Reply:
לא אופיין באמת, להוסיף בכל זאת?

Reply:
could help to debug if tab doesnt appear

Reply:
no idea

Reply:
ma she ata hoshev

Reply:
הוספתי